src/redux/books/books.js

```diff
@@ -0,0 +1,39 @@
+
+const ADDBOOK = "ADDBOOK";
+const REMOVEBOOK = "REMOVEBOOK";
```

Comment:
- [ ] Good job so far 👍 The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like this: `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`. Kindly rectify this issue 👍

Reply:
Thanks a lot for this. I will do it. But I am a bit confused. Could you please tell me what's the difference between these two approaches? actually, I am a bit confused regarding ducks pattern.
I would be very thankful if you clear up this confusion.